templates/user/my_donation.html.twig

```diff
@@ -54,10 +54,11 @@
                 </tr>
             </thead>
             <tbody>
-                {% for donation in donations %}
+                {% for transaction in transactions_success %}
+                    {% set donation = transaction.donation %}
```

Comment:
can you check with profiler of SF how many db queries you execute when you are in this view?
I think you need improve your dql for select donation too.

Reply:
forgot addSelect :D